website/content/docs/oss/configuration/kms/ocikms.mdx

```diff
@@ -44,7 +44,7 @@ These parameters apply to the `kms` stanza in the Boundary configuration file:
 Authentication-related values must be provided, either as environment
 variables or as configuration parameters.
 
-If you want to use Instance Principal, add section configuration below and add further configuration settings as detailed in the [configuration docs](/docs/configuration/).
+If you want to use Instance Principal, add section configuration below and add further configuration settings as detailed in the [configuration docs](/docs/oss/configuration/).
```

Comment:
Just a meta-note. It would be really good to figure out a way to share docs between Vault and Boundary for KMS as they use the same library and there's no real reason the docs should get divergent (especially once Vault updates to v2 of the lib).